patroni/etcd.py

```diff
@@ -49,12 +50,57 @@ def machines(self):
             self._update_machines_cache = True
             return [self._base_uri]
 
-    def api_execute(self, path, method, **kwargs):
+    def _do_http_request(self, request_executor, method, url, fields=None, **kwargs):
+        try:
+            response = request_executor(method, url, fields=fields, **kwargs)
+            response.data.decode('utf-8')
+#            self._check_cluster_id(response)
+        except (urllib3.exceptions.HTTPError, HTTPException, socket.error) as e:
+            if (isinstance(fields, dict) and fields.get("wait") == "true" and
+                    isinstance(e, urllib3.exceptions.ReadTimeoutError)):
+                logger.debug("Watch timed out.")
+                raise etcd.EtcdWatchTimedOut("Watch timed out: %s".format(e), cause=e)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Too many arguments for format string](https://www.codacy.com/app/cyberdemn/patroni/file/2229167422/issues/source?bid=3127594&fileBranchId=3175390#l62)